modules/mono/editor/script_templates/VisualShaderNodeCustom/basic.cs

```diff
@@ -3,6 +3,7 @@
 using _BINDINGS_NAMESPACE_;
 using System;
 
+[Godot.Tool]
```

Comment:
```suggestion
[Tool]
```
Other templates don't use `Godot.Tool` but `Tool`, so shouldn't be used here either I think

Reply:
Yep. By the way `using _BINDINGS_NAMESPACE_;` is converted to `using Godot;` which is why the type name doesn't need to be fully-qualified.

Reply:
Great!
Yeah, I figured. But thought better safe than sorry, haha!